jetstream/metadata.py

```diff
@@ -15,9 +15,10 @@
 
 @attr.s(auto_attribs=True)
 class MetricsMetadata:
-    friendly_name: str
-    description: str
+    friendly_name: Optional[str]
+    description: Optional[str]
```

Comment:
This seems realistic; I'm not sure how this typed before :D but are we sure that Experimenter can accept missingness here? @emtwo

Reply:
Looking at the Experimenter code [here](https://github.com/mozilla/experimenter/blob/258574f9f90abae0af292f51efc5cbe8f9fcd102/app/experimenter/nimbus-ui/src/lib/visualization/types.ts#L23) and [here](https://github.com/mozilla/experimenter/blob/c121f12957ff79e9233d4de4ac83851b1a0fcbf9/app/experimenter/nimbus-ui/src/components/AppLayoutSidebarLaunched/index.tsx#L48) it appears both are expected to be there

Reply:
I changed the types here to `str` and made sure if values are missing they are replaced by empty strings so that Experimenter doesn't break

Reply:
🌈 perfect thanks